lib/ansible/modules/storage/zfs/zfs.py

```diff
@@ -34,12 +34,6 @@
   origin:
     description:
       - Snapshot from which to create a clone.
-  key_value:
-    description:
```

Comment:
this was not properly deprecated, there is a specific process to follow, a deprecation field in the documentation (not just adding a line in the description), but since at least the runtime use of the option did issue a proper deprecation warning, Im in favor of letting that go.